go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/docker/docker v20.10.3+incompatible
 	github.com/docker/go-connections v0.4.0 // indirect
 	github.com/docker/go-units v0.4.0 // indirect
+	github.com/go-git/go-git/v5 v5.4.2
```

Comment:
hmm, why was go-git added. I didn't see any of the API's being used. this is the direction I would like to go tho 😆

Reply:
Will remove the import for now